File: android/app/src/main/java/com/npkompleet/beatz/models/AudioMedia.java
package com.npkompleet.beatz.models;

public class AudioMedia {
    private int id;
    private String title;
    private String displayName;
    private String artist;
    private long duration;
    private int track;

    public AudioMedia() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDisplayName() {
        return displayName;
    }

    public void setDisplayName(String displayName) {
        this.displayName = displayName;
    }

    public String getArtist() {
        return artist;
    }

    public void setArtist(String artist) {
        this.artist = artist;
    }

    public long getDuration() {
        return duration;
    }

    public void setDuration(long duration) {
        this.duration = duration;
    }

    public int getTrack() {
        return track;
    }

    public void setTrack(int track) {
        this.track = track;
    }
}

File: lib/models/audio_media.dart
class AudioMedia {
  int id;
  String title;
  String displayName;
  String artist;
  int duration;
  int track;
  String uri;
  String type;

  AudioMedia(this.id, this.title, this.displayName, this.artist, this.duration,
      this.track, this.uri, this.type);

  AudioMedia.fromJson(Map<String, dynamic> json) {
    artist = json['artist'];
    displayName = json['displayName'];
    duration = json['duration'];
    id = json['id'];
    title = json['title'];
    track = json['track'];
    uri = json['uri'];
    type = json['type'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['artist'] = this.artist;
    data['displayName'] = this.displayName;
    data['duration'] = this.duration;
    data['id'] = this.id;
    data['title'] = this.title;
    data['track'] = this.track;
    data['uri'] = this.uri;
    data['type'] = this.type;
    return data;
  }
}
